jscomp/runtime/js_typed_array.cppo.ml

```diff
@@ -54,123 +49,17 @@ module ArrayBuffer = struct
 
   external byteLength : t -> int = "byteLength" [@@mel.get]
 
-  external slice : start:int -> end_:int -> array_buffer = "slice" [@@mel.send.pipe: t] (*FIXME*)
-  external sliceFrom : int -> array_buffer = "slice" [@@mel.send.pipe: t]
+  external slice : t -> ?start:int -> ?end_:int -> unit -> array_buffer = "slice" [@@mel.send]
 end
 
 open struct
   module Js = Js_internal
 end
 
-module type S =  sig
```

Comment:
What was this used for?

Reply:
no idea. The way I've been handling these migrations has been to move `foo2.ml` to `foo.ml`. So this wasn't present in `Js_typed_array2`. I don't think we need to keep it.